src/packages/core/components/input-image-cropper/image-cropper-focus-setter.element.ts

```diff
@@ -1,4 +1,5 @@
-import { clamp } from './mathUtils.js';
+//import { clamp } from './mathUtils.js';
+import { drag, clamp } from '@umbraco-ui/uui-base/lib/utils';
```

Comment:
Currently backoffice doesn't have a `drag` method and `clamp` is a copy from what already exists in UI library.
Should the clamp function be removed from backoffice repository or just re-use functions from UI library?

Reply:
No, you should be able to re-export them from the `external/uui` library and import it directly here:
```js
import { drag, clamp } from '@umbraco-cms/backoffice/external/uui';
```

Reply:
I have updated this in https://github.com/umbraco/Umbraco.CMS.Backoffice/pull/1021/commits/c2c7b48e7e5c0682861fda5e46841255f32e50d7